src/client/runner/automation/playback/scroll.js

```diff
@@ -36,12 +36,13 @@ export default class ScrollAutomation {
 
     static _setScroll (element, { left, top }) {
         if (domUtils.isDocumentRootElement(element)) {
-            styleUtils.setScrollLeft(document.body, left);
-            styleUtils.setScrollTop(document.body, top);
+            styleUtils.setScrollLeft(document.body, Math.max(left, 0));
```

Comment:
please, make rebase to avoid this changes that were implemented in the [fix unstable tests in IOS](https://github.com/DevExpress/testcafe/pull/259/files) issue